addons/web_tour/static/src/js/tip.js

```diff
@@ -232,36 +342,39 @@ var Tip = Widget.extend({
 
         var offset = this.$el.offset();
 
-        if (this.$el.parent()[0] !== document.body) {
-            this.$el.detach();
-            this.$el.css(offset);
-            this.$el.appendTo(document.body);
-        }
+        if (offset.top || offset.left) { // Resolve cases when the info box is triggered before the tip is fully attached
```

Comment:
Seems strange, can you explain this better to me? Why was not it breaking before your branch? Also, instead of indenting the rest of the function for nothing, can't you just exit the function if the condition is not satisfied?

Reply:
Ok, I dig a bit more into, It seems like when I consume a step and open it in the same time, it could try to open the one you just consume.
If you click in the same time that you hover the anchor, could create this bug. (And could be easier than expected when the browser is a bit laggy ?)
It's like the info bubble had the time to add its elements in the DOM before the code destroy the tip (and so, the DOM added couldn't be removed without refreshing the page because the tip and its methods are destroyed)

Not 100% sure of my hypothesis but seems the more plausible .

I will see if I don't have a better solution around the destroy method.

Reply:
So, this.$el doesn't exist anymore in the DOM and it's why the bubble goes to 0;0. 
What I can do, it's when I create a bubble, to make sure that the $el still exist in the DOM, I checked it, and it doesn't have any parent in this case (which is cleaner than check its position).
I don't know if this bug was there before my modifications or not. Tried to reproduce it in master with the banner tour but no results.

I tried to find its provenance, but didn't find any other corrections than this one.

Reply:
Mouais, did not understand. Change as you think it should be and at least do not indent the whole function like you did before, then I'll check.